docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -130,14 +130,30 @@ NOTE: Since eland uses APIs to deploy the models, you cannot see the models in
 {kib}, wait for automatic synchronization, or use the
 {kibana-ref}/machine-learning-api-sync.html[sync {ml} saved objects API].
 
-When you deploy the model, it is allocated to all available {ml} nodes. The
-model is loaded into memory in a native process that encapsulates `libtorch`,
-which is the underlying machine learning library of PyTorch.
-
-TIP: You can set additional deployment options that affect the performance of
-the model. For example, specify the number of allocations and the threads per
-allocation to optimize the throughput and speed of inference requests in your
-production environment. For more information about these options, refer to the
+When you deploy the model, its allocations are distributed across available {ml} 
+nodes. Model allocations are independent units of work for NLP tasks. Throughput 
+can be scaled by adding more allocations to the deployment, it increases the 
+number of {infer} requests that can be performed in parallel. All allocations 
+assigned to a node share the same copy of the model in memory. The model is 
+loaded into memory in a native process that encapsulates `libtorch`, which is 
+the underlying {ml} library of PyTorch.
+
+You can set the number of allocations and the number of threads an allocation 
```

Comment:
> You can set the number of allocations and the number of threads an allocation 

I think you used to only be able to specify these settings via the API. If that's still true, it should be mentioned here

Reply:
Yup, thanks! I'll remove the `8.3.1` and the backport label.